docker-compose.yml

```diff
@@ -76,6 +77,11 @@ services:
       - REFORM_ENVIRONMENT=aat
       - REFORM_SERVICE_NAME=payment-api
       - REFORM_TEAM=cc
+      - SERVICE_REQUEST_REPORT_SCHEDULER_ENABLED=true
+      - SERVICE_REQUEST_EMAIL_FROM=no-reply@reform.hmcts.net
+      - SERVICE_REQUEST_EMAIL_TO=pepito@grillo.com
+      - SERVICE_REQUEST_EMAIL_MESSAGE=hi
+      - SERVICE_REQUEST_EMAIL_SUBJECT=Docker Env Service Request Report
```

Comment:
Can you refactor these SERVICE_REQUEST_XXX environment variables, so you have some more descriptive for it's intended use, I.e. DUPLICATE_SERVICE_REQUEST_XXX.